parts-list-specs/updated-parts-list-spec.R

```diff
@@ -0,0 +1,24 @@
+
+[partLabel]([partID]): [partDesc] [partInstr; if == NA, print = FALSE]
+"Part Type:" [partType]
+"Domain ID:" [domainID]
+"Specimen ID:" [specimenID]
+"Compartment Set:" [compartSet]
```

Comment:
compSet?

Reply:
@rvyuha Can you update this line and then merge it into master?